assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/workers/variables.tf

```diff
@@ -94,8 +94,13 @@ variable "cluster_domain_suffix" {
   default     = "cluster.local"
 }
 
-variable "kubeconfig" {
-  description = "Kubeconfig file"
+variable "ca_cert" {
```

Comment:
Must this be base64 encoded? Or should we base64 encode it in the template?

Reply:
It is already done here https://github.com/kinvolk/lokomotive/blob/e2499b44b06aadd76cd944250a9cdfa2d104ec93/assets/lokomotive-kubernetes/bootkube/outputs.tf#L49-L51

Reply:
Ah, thanks :+1: